scheduler/src/cook/components.clj

```diff
@@ -291,6 +292,13 @@
                                   offer-incubate-ms)
      :mesos-master (fnk [[:config [:mesos master]]]
                         master)
+
+     :mesos-master-hosts (fnk [[:config [:mesos master {master-hosts nil}]]]
```

Comment:
This is exposed in end-user config. User can specify master-hosts in the config.